app/assets/stylesheets/chat-ui-common.scss

```diff
@@ -231,3 +231,8 @@ $footer-top-border-height: 3px;
         display: inline-block;
     }
 }
+
+#sentMessage {
+    position: absolute;
+    left: -9999px
```

Comment:
Whats this for?

Reply:
This positions the #sentMessage which is created in the onMessageSentNotification method off screen so it does not interfere with the display. What do you think?